go/tasks/plugins/webapi/athena/plugin.go

```diff
@@ -184,6 +188,14 @@ func createTaskInfo(queryID string, cfg awsSdk.Config) *core.TaskInfo {
 				Name: "Athena Query Console",
 			},
 		},
+		Metadata: &event.TaskExecutionMetadata{
+			PluginIdentifier: athenaPluginName,
```

Comment:
I think the PluginIdentifier should be injected by the TaskHandler itself in FlytePropeller. This is error prone

Reply:
if not in propeller then the WebAPI plugin, PluginManager should inject it. But the simplest would be to inject it in the transformer for task/handler.go

Reply:
done, added to propeller